assets/queries/ansible/aws/route53_record_without_values/query.rego

```diff
@@ -0,0 +1,32 @@
+package Cx
+
+CxPolicy[ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  valueIsEmpty(task["community.aws.route53"].value)
+  
+  result := {
+      "documentId": document.id,
+      "searchKey": sprintf("name=%s.{{community.aws.route53}}.value", [task.name]),
+      "issueType": "IncorrectValue",
+      "keyExpectedValue": "aws_route53.value is not empty",
+      "keyActualValue": "aws_route53.value is empty"
+  }
+}
+
+getTasks(document) = result {
+  result := [body | playbook := document.playbooks[0]; body := playbook.tasks]
+  count(result) != 0
+} else = result {
+  result := [body | playbook := document.playbooks[_]; body := playbook ]  
+  count(result) != 0
+}
+
+valueIsEmpty(value) {
+	value == null
+}
+valueIsEmpty(value) {
+	value != null
+  count(value) <= 0
+} 
```

Comment:
The Parameter 'value' is not required. We should check first if it exists, create one result for when it's missing